src/main/cms/cms_menu_gps_rescue.c

```diff
@@ -205,7 +204,7 @@ const OSD_Entry cmsx_menuGpsRescueEntries[] =
     { "THROTTLE MAX",      OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &gpsRescueConfig_throttleMax, 1000, 2000, 1 } },
     { "THROTTLE HOV",      OME_UINT16 | REBOOT_REQUIRED, NULL, &(OSD_UINT16_t){ &gpsRescueConfig_throttleHover, 1000, 2000, 1 } },
 
-    { "SATS REQUIRED",     OME_UINT8 | REBOOT_REQUIRED, NULL, &(OSD_UINT8_t){ &gpsConfig_gpsRequiredSats, 4, 50, 1 } },
+    { "SATS REQUIRED",     OME_UINT8 | REBOOT_REQUIRED, NULL, &(OSD_UINT8_t){ &gpsRescueConfig_minSats, 5, 50, 1 } },
```

Comment:
```suggestion
    { "SATS REQUIRED",     OME_UINT8 | REBOOT_REQUIRED, NULL, &(OSD_UINT8_t){ &gpsRescueConfig_minSats, GPS_MIN_SAT_COUNT, 50, 1 } },
```

Is the change to an "allegedly random" hardcoded value wanted? How about this (or `GPS_MIN_SAT_COUNT + 1`)?

Reply:
Hmm... good point, gotta think about it a bit.

Reply:
...or you could also `#define` a new macro, if it is independent of `GPS_MIN_SAT_COUNT`. Just to give this constant a name for transparency and maintainability, I guess.

Reply:
Maybe.  But I think we may well change this in 4.5 when we look closer at position accuracy indexes.  

The value chosen needs to provide some headroom against sanity check failures after taking off with only just `GPS_MIN_SAT_COUNT` sats.  Usually you get a few more as you fly around, but not always.  With the default of 8 I think the min is somewhat arbitrary.  

It's easiest for now to leave it as is, so maybe when were look closer at alternative position accuracy estimates, we may want to re-purpose this, so perhaps lets just leave it as is.